### DISCUSSION TITLE
Running Fedora ARM on M-series from RAW image?

### DISCUSSION THREAD

**Discussion by taoeffect** [utmapp/UTM#6575] [UTM Help (macOS)]
**Describe the issue**  

For whatever reason, Fedora doesn't provide ISO files for ARM:

- https://fedoraproject.org/workstation/download
- https://fedoraproject.org/spins/kde/download

I'd like to install the latest Fedora (40) KDE version, but it just gives a .raw.xz file.

Attempting to start a virtual machine using the decompressed file (.raw.xz => .raw) doesn't work.

What to do?

**Configuration**  
* UTM Version: 4.5.3 (99)
* macOS Version: 14.5
* Mac Chip (Intel, M1, ...): M3

**Comment by dmichael89** [comment]
You could try the server version at https://alt.fedoraproject.org/alt/

**Comment by taoeffect** [comment]
That doesn't work either, try it: https://download.fedoraproject.org/pub/fedora-secondary/releases/40/Spins/aarch64/iso/Fedora-KDE-Live-aarch64-40-1.14.iso

```
error: invalid path
```

**Comment by dmichael89** [comment]
On my iPad in both cases I get a valid URL.Von meinem iPad gesendetAm 01.06.2024 um 20:01 schrieb Greg Slepak ***@***.***>:That doesn't work either, try it: https://download.fedoraproject.org/pub/fedora-secondary/releases/40/Spins/aarch64/iso/Fedora-KDE-Live-aarch64-40-1.14.iso
error: invalid path

**Comment by taoeffect** [comment]
you're able to download it?

**Comment by dmichael89** [comment]
I‘ve downloaded and installed the server versionVon meinem iPhone gesendetAm 01.06.2024 um 21:41 schrieb Greg Slepak ***@***.***>:﻿
you're able to download it?

**Comment by taoeffect** [comment]
I have tried to download that ISO from that link from multiple locations and the link returns a 404 for all of them. I don't understand how you could have downloaded from this link: https://download.fedoraproject.org/pub/fedora-secondary/releases/40/Spins/aarch64/iso/Fedora-KDE-Live-aarch64-40-1.14.iso

The raw file does download, but for that I can't get UTM to run it.

**Comment by dmichael89** [comment]
as I told you, I use the server version Von meinem iPhone gesendetAm 01.06.2024 um 22:07 schrieb Greg Slepak ***@***.***>:﻿
I have tried to download that ISO from that link from multiple locations and the link returns a 404 for all of them. I don't understand how you could have downloaded from this link: https://download.fedoraproject.org/pub/fedora-secondary/releases/40/Spins/aarch64/iso/Fedora-KDE-Live-aarch64-40-1.14.iso
The raw file does download, but for that I can't get UTM to run it.

**Comment by SConaway** [comment]
You can use the Everything netinstall image: https://download.fedoraproject.org/pub/fedora/linux/releases/40/Everything/aarch64/iso/Fedora-Everything-netinst-aarch64-40-1.14.iso (found at: https://alt.fedoraproject.org/alt/). That seems to work just fine for installing the KDE desktop (or others), might take a bit longer to install since the ISO doesn't contain much.

**Comment by taoeffect** [comment]
Hmm, thanks! That seems to work! Sortof... stuck here at "error setting up repositories" and not sure what to do.

![Screenshot 2024-06-10 at 7 16 09 PM](https://github.com/utmapp/UTM/assets/138706/d1b31877-406d-4ee4-b053-e222564cc6cc)

![Screenshot 2024-06-10 at 7 16 16 PM](https://github.com/utmapp/UTM/assets/138706/c0571f3a-febb-4280-9473-002f58ce1440)

**Comment by SConaway** [comment]
By chance are you using bridged networking? I’ve had a bunch of issues with that, two different networks. Apple virtualization or QEMU?

**Comment by taoeffect** [comment]
I'm using a shared network. I tried with both Apple virtualization and QEMU and got the same results with both.

**Comment by ideologysec** [comment]
@taoeffect you can use the RAW images as provided by Fedora, but you need a workaround.

1. create the VM in UTM, point it to a fake or non-fedora ISO for boot (even `touch ~/Downloads/Fedora.iso` creating an empty file worked for me)

2. check the box for "Open VM Settings" and REMOVE both drives 

<img width="187" alt=" 2024-06-11 at 12 45 18" src="https://github.com/utmapp/UTM/assets/790061/34b641d1-4944-4341-916a-5928b18c16b3">
<img width="178" alt=" 2024-06-11 at 12 45 41" src="https://github.com/utmapp/UTM/assets/790061/050b784b-fd97-42a0-88b7-6dbf9842a118">

3. select "New" under drives and click "Import" and then point it at the Fedora img (you need to un-xz it), then save. UTM will copy into the VM.

4. start the VM, it should present you with user setup.

<img width="795" alt=" 2024-06-11 at 12 58 16" src="https://github.com/utmapp/UTM/assets/790061/fc160adb-04fa-419f-83e3-e12db3898bc0">

<img width="1024" alt=" 2024-06-11 at 12 59 19" src="https://github.com/utmapp/UTM/assets/790061/5a1129fc-aa15-4be5-b2dd-300b3f818459">

**Comment by ideologysec** [comment]
#6308 is related to this.

**Comment by SConaway** [comment]
@taoeffect when using the ISO, what happens if you select closest mirror: 


![UTM - 2024-06-11 03 24 43 PM](https://github.com/utmapp/UTM/assets/20807660/e4718982-d5ba-4a4c-9d7c-b1a01c9c81bb)

It works just fine for me, and I've created a few VMs using this image.

**Comment by bosshoss16** [comment]
> @taoeffect you can use the RAW images as provided by Fedora, but you need a workaround.

I was able to create a VM using this method.  However, as soon as I updated Fedora, it would not boot, anymore.  I am sure the errors are for a separate issue, but I wanted to say it worked for me.
